packages/website/src/widgets/header/header.test.tsx

```diff
@@ -1,5 +1,5 @@
 import "@testing-library/jest-dom";
-import { render, screen } from "@testing-library/react";
+import { render, screen } from "@/shared/lib/utilities/testing";
```

Comment:
Why did we need to make testing utilities? I'm having trouble following the render and createWrapper logic 😅

Reply:
The purpose of this is to support components and logic that rely on routing. In the future, this will be enhanced to include the data store.

Thus, instead of wrapping our components in a router manually over and over again, these utilities will do them for us.